### PR TITLE
Closes #3124: Unable to upload images

### LINKED ISSUE #3124: image-attachments API POST Request: Unsupported media type "multipart/form-data"
### Environment
* Nautobot version (Docker tag too if applicable): 1.5.7
* Python version: 3.8.10
* Database platform, version: postgresql, Ubuntu 12.12-0ubuntu0.20.04.1
* Middleware(s): - 

### Steps to Reproduce
1. Upload a jpg file via api POST request and "multipart/form-data" content type to https://nautobot-server/api/extras/image-attachments/ 

### Expected Behavior
The image will be uploaded like in 1.4.x releases

### Observed Behavior
Upload is not possible anymore and cancels with the error message "Unsupported media type "multipart/form-data; boundary="xxxxxxx""

According to the api documentation only "application/json" is available in v1.5.7. With older nautobot versions multipart/form-data was available. The API POST command for image-attachments is useless without multipart/form-data as content-type.

### PR DESCRIPTION
**Pull request by austind**
# Closes #3124 
# What's Changed

The `DeviceTypeViewSet` and `ImageAttachmentViewSet` now include the `MultiPartParser` class, which allows uploading images via API.
# Screenshots

# TODO

- [x] Explanation of Change(s)
- [x] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [x] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example App Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Comment by austind** [comment]
> I'm assuming you've tested this locally? Looks good to me if so. Thanks!

Yep, this now works:

```python
url = 'http://localhost:8080/api/dcim/device_types/42fc565d-7fad-4bea-b492-3995ce2a8585'
token = '0123456789abcdef0123456789abcdef01234567'
images = {
    'front_image': Path('path/to/image.png'),
    'rear_image': Path('path/to/image.png'),
}
files = {i: (f.name, f.read_bytes()) for i, f in images.items()}
headers = {"Authorization": f"Token {token}"}

response = requests.patch(
    url=url,
    files=files,
    headers=headers,
    verify=False,
    timeout=10.0,
)
```

File: nautobot/extras/api/views.py [hunk 1]
```diff
@@ -484,6 +484,7 @@ class ImageAttachmentViewSet(ModelViewSet):
     queryset = ImageAttachment.objects.all()
     serializer_class = serializers.ImageAttachmentSerializer
     filterset_class = filters.ImageAttachmentFilterSet
+    parser_classes = [JSONParser, MultiPartParser]
```
Comment (glennmatthews) [inline nautobot/extras/api/views.py:487]: I don't think we support CSV export of image attachments, but including `NautobotCSVParser` here might be a good idea regardless.

Comment (austind) [reply nautobot/extras/api/views.py]: Done!

Comment (austind) [reply nautobot/extras/api/views.py]: Actually, looks like that broke CI?

Comment (glennmatthews) [reply nautobot/extras/api/views.py]: No, that failure's an intermittent one that we haven't been able to root-cause yet. I just kicked off a rerun which will probably pass.

**Review by HanlinMiao — APPROVED** [review]

**Pull request merged**